README.adoc

```diff
@@ -73,6 +73,22 @@ To dump the state of the wallet in `~/bitcoinj/bitcoinj-test.wallet` with the te
 
 NOTE: These instructions are for macOS/Linux, for Windows use the `wallettool/build/install/wallet-tool/bin/wallet-tool.bat` batch file with the equivalent Windows command-line commands and options.
 
+### Building and Running the Wallet Template
+
+The *bitcoinj* `wallettemplate` subproject includes a template JavaFX wallet application (`bitcoinj-wallettemplate`) that can be used as a starting point for building a JavaFX-based *bitcoinj* wallet application.
+
+To build an executable shell script that runs the wallettemplate, use:
+```
+gradle bitcoinj-wallettemplate:installDist
+```
+
+You can now run the `bitcoinj-wallettemplate` to launch the application:
+```
+./wallettemplate/build/install/bitcoinj-wallettemplate/bin/bitcoinj-wallettemplate
+```
+
+NOTE: These instructions are for macOS/Linux, for Windows use the `wallettemplate/build/install/bitcoinj-wallettemplate/bin/bitcoinj-wallettemplate.bat` batch file to start the application.
```

Comment:
I'll remove these "running instructions" from this commit because this commit doesn't fix the running. PR #3566 does, we should add them to there.

I think I'd just make two bullet points:

- Linux, macOS: …
- Windows: …

Or alternatively, not mention Windows at all (-:

Reply:
I realized that after preparing everything. I was hoping you were going to let it slide...

Reply:
> Or alternatively, not mention Windows at all (-:

How about if I just shorten the NOTE considerably? Stay tuned...